### PR TITLE
Merge no-wearable-recording into main

### PR DESCRIPTION
**Pull request by mdmohsin7**
Related to #348

### CODE REVIEW

File: apps/AppWithWearable/ios/Runner.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -711,6 +711,7 @@
 				CLANG_WARN_QUOTED_INCLUDE_IN_FRAMEWORK_HEADER = YES;
 				CLANG_WARN_UNGUARDED_AVAILABILITY = YES_AGGRESSIVE;
 				CODE_SIGN_ENTITLEMENTS = OneSignalNotificationServiceExtension/OneSignalNotificationServiceExtension.entitlements;
+				CODE_SIGN_IDENTITY = "Apple Development";
```
Comment (francip) [inline apps/AppWithWearable/ios/Runner.xcodeproj/project.pbxproj:740]: Can you revert those changes? The are not needed for your PR.

File: apps/AppWithWearable/ios/Runner/Info.plist [hunk 2]
```diff
@@ -6,6 +6,7 @@
 		<array>
 			<string>daily-summary</string>
 			<string>com.friend-app-with-wearable.ios12.daily-summary</string>
+      <string>dev.flutter.background.refresh</string>
```
Comment (francip) [inline apps/AppWithWearable/ios/Runner/Info.plist]: [nitpick] indent :-)

File: apps/AppWithWearable/ios/Runner.xcodeproj/project.pbxproj (hunk 1 above)
Comment (mdmohsin7) [reply apps/AppWithWearable/ios/Runner.xcodeproj/project.pbxproj:740]: I'm not really sure what changes caused those values to be added to the pbxproj file. So, can't really revert those changes. I guess Joan running the app locally will override these changes

**Review by josancamon19 — APPROVED** [review]
Merging for now, but still has bugs, merging now to avoid more merge conflicts, bugs sent through discord.

**Pull request merged**